.github/workflows/codeql-analysis.yml

```diff
@@ -22,10 +22,71 @@ jobs:
       fail-fast: false
       matrix:
         language: [ 'cpp', 'csharp' ]
+        arch: [x64, arm64]
```

Comment:
Do we need to run this on arm64 too? I wouldn't think the CodeQL analysis depended much on the architecture unless there was a lot of architecture specific code, so maybe just x64 is fine